json-endpoints/jetpack/class.jetpack-json-api-themes-endpoint.php

```diff
@@ -100,7 +101,8 @@ protected function format_theme( $theme ) {
 		$id = $theme->get_stylesheet();
 		$formatted_theme = array(
 			'id'          => $id,
-			'screenshot'  => jetpack_photon_url( $theme->get_screenshot(), array(), 'network_path' )
+			'screenshot'  => jetpack_photon_url( $theme->get_screenshot(), array(), 'network_path' ),
+			'active'      => $id === $this->current_theme_id,
```

Comment:
In the future, when it will be possible to install themes from dotcom -- will prefixes like `pub/` or `premium/` present in the return value of `get_stylesheet()` (L157) screw up this comparison?